### PR TITLE
Policy admin/feature/hl 72114 get knex clone schema injections lat

### PR DESCRIPTION
**Pull request by srksag**

### CODE REVIEW

File: components/persistor/HISTORY.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
 ## 9.0.2
```
Comment (kamranHaven) [inline components/persistor/HISTORY.md]: I believe you meant for this to be 9.1.0

File: components/amorphic/lib/startApplication.js [hunk 2]
```diff
@@ -35,8 +35,19 @@ function startApplication(appName, appDirectory, appList, configStore, sessionSt
     if (config.serverMode === 'daemon' || config.serverMode === 'api') {
         controllerJsDir = path + '/js/';
     }
-
-    let schema = JSON.parse((readFile(path + '/schema.json') || readFile(commonPath + '/schema.json')).toString());
+    
+    let schema = null;
+    if (externalSchemas) {
+        schema = externalSchemas;
+    }
```
Comment (rognorak) [inline components/amorphic/lib/startApplication.js]: copy pasta?  should this be a method / function

File: components/amorphic/lib/unitTestConfig.js [hunk 3]
```diff
@@ -24,8 +24,18 @@ function startup(configPath, schemaPath, configStore = null) {
 
     config.nconf = configStore['root']; // Global config
     config.configStore = configStore;
-
-    let schema = JSON.parse((readFile(schemaPath + '/schema.json')).toString());
+    let schema = null;
+    if (externalSchemas) {
+        schema = externalSchemas;
```
Comment (rognorak) [inline components/amorphic/lib/unitTestConfig.js]: does schemas support an array, for more than one module to inject a schema?  Do we have a unit test for that case?  What happens if 2  eternal schemas define the same table ?

File: components/persistor/HISTORY.md (hunk 1 above)
Comment (srksag) [reply components/persistor/HISTORY.md]: Good catch - fixed it.

File: components/amorphic/lib/startApplication.js (hunk 2 above)
Comment (srksag) [reply components/amorphic/lib/startApplication.js]: Mike - one scenario is using common folder as well and in other scenario, we are only loading from the path provided. So I left the code as it's added the injection logic.

**Review by kamranHaven — DISMISSED** [review]

File: components/amorphic/lib/startApplication.js (hunk 2 above)
Comment (srksag) [reply components/amorphic/lib/startApplication.js]: Refactored the code

**Review by kamranHaven — APPROVED** [review]

**Review by rognorak — APPROVED** [review]

**Pull request merged**